### PR TITLE
V2X: v2x perception fusion

### PR DESCRIPTION
**Pull request by WhitingHuo**

### CODE REVIEW

File: modules/v2x/fusion/libs/fusion/fusion.cc [hunk 1]
```diff
@@ -106,6 +106,39 @@ bool Fusion::CombineNewResource(
   return true;
 }
 
+bool Fusion::GetV2xFusionObjects(
+    const std::vector<std::vector<base::Object>> &fusion_result,
+    std::vector<base::Object> *fused_objects) {
+  for (auto &objects : fusion_result) {
```
Comment (changsh726) [inline modules/v2x/fusion/libs/fusion/fusion.cc:112]: const auto &

File: modules/v2x/fusion/libs/fusion/fusion.cc [hunk 2]
```diff
@@ -106,6 +106,39 @@ bool Fusion::CombineNewResource(
...
+      if (objects.at(0).frame_id == "V2X") {
+        fused_objects->back().v2x_type = base::V2xType::BLIND_ZONE;
+      } else {
+        fused_objects->back().v2x_type = base::V2xType::UNKNOWN;
+      }
+    } else {
+      fused_objects->push_back(objects.at(0));
+      host_vehicle_ = false;
+      zom_vehicle_ = false;
+      for (auto &object : objects) {
```
Comment (changsh726) [inline modules/v2x/fusion/libs/fusion/fusion.cc:124]: Ditto.

File: modules/v2x/fusion/apps/common/trans_tools.cc [hunk 3]
```diff
@@ -118,10 +118,9 @@ void Pb2Object(const PerceptionObstacle &obstacle, base::Object *object,
 }
 
 void V2xPb2Object(const apollo::v2x::V2XObstacle &obstacle,
-                   base::Object *object, const std::string &frame_id,
-                   double timestamp_object) {
-  Pb2Object(obstacle.perception_obstacle(), object, frame_id,
-             timestamp_object);
+                  base::Object *object, const std::string &frame_id,
+                  double timestamp_object) {
```
Comment (lx18233184051) [inline modules/v2x/fusion/apps/common/trans_tools.cc:122]: const double& timestamp_object ?

File: modules/perception/production/dag/dag_streaming_perception.dag [hunk 4]
```diff
@@ -67,5 +67,17 @@ module_config {
         }
     }
   }
+  components {
```
Comment (changsh726) [inline modules/perception/production/dag/dag_streaming_perception.dag:70]: Leave a blank line here.

**Review by changsh726 — CHANGES_REQUESTED** [review]

File: modules/v2x/fusion/apps/common/trans_tools.cc (hunk 3 above)
Comment (WhitingHuo) [reply modules/v2x/fusion/apps/common/trans_tools.cc:122]: 对象属于内置类型，按值传递往往比按引用传递效率高

**Review by changsh726 — APPROVED** [review]

**Pull request merged**